staging/src/k8s.io/kube-aggregator/go.mod

```diff
@@ -49,6 +49,7 @@ require (
 	github.com/google/cel-go v0.12.6 // indirect
 	github.com/google/gnostic v0.5.7-v3refs // indirect
 	github.com/google/uuid v1.3.0 // indirect
+	github.com/gorilla/websocket v1.4.2 // indirect
```

Comment:
@mpuckett159 already mentioned the gorilla project is archived are we going to use this package anyway?

Reply:
all websocket packages out there are very stable, is not much things to add to this stable protocol, if we found a bug with it we'll need to revisit this decision though